templates/rhel8.tpl.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: {{ os }}-{{ item.workload }}-{{ item.flavor }}-v0.7.0
```

Comment:
Please update the name suffix to the actual version that will first contain this change.